sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,507 @@
+            ''--TBD--'' as "Reason for F. Student Contr. Exemp.: PD",
+            ''--TBD--'' as "Reason for F. Student Contr. Exemp.: Dependants",
+            ''--TBD--'' as "Reason for F. Student Contr. Exemp.: Cur. or Former Crown Ward",
+            ''--TBD--'' as "Reason for F. Rate Spousal Contr. Exemp.: Student",
+            ''--TBD--'' as "Reason for F. Rate Spousal Contr. Exemp.: Rec. Employment Ins.",
+            ''--TBD--'' as "Reason for F. Rate Spousal Contr. Exemp.: Rec. Social Assist.",
+            ''--TBD--'' as "Reason for F. Rat. Spo. Contr. Exemp.: Rec. fed./prov. dis. ben",
+            ''--TBD--'' as "Fixed student contribution review flag",
+            ''--TBD--'' as "parental contribution review flag",
+            ''--TBD--'' as "Fixed spousal contribution review flag",
+            -- Application Processing Info
+            applications.application_number as "Application Number",
+            ''--TBD--'' as "Version Number",
+            applications.application_status as "Application Status",
+            to_char(
+                student_assessments.assessment_date,
+                ''YYYY-MM-DD''
+            ) as "Assessment Date",
+            student_assessments.trigger_type as "Re-assessment Indicator",
+            ''--TBD--'' as "CSL Loan authorization date",
+            student_assessments.workflow_data -> ''studentData'' ->> ''relationshipStatus'' as "Marital Status (or ''category'')",
+            student_assessments.workflow_data -> ''studentData'' ->> ''dependantStatus'' as "Student Dependency Status",
+            ''--TBD--'' as "Reason for Single Independant Status",
+            -- Verification Info
+            (
+                (
+                    select
+                        count(1)
+                    from
+                        sims.applications a
+                    where
+                        a.application_number = applications.application_number
+                ) -1 --Number of editions after the first input
+            ) as "Edit history",
+            ''--TBD--'' as "Number of edits to income fields",
+            (
+                select
+                    a.data ->> ''taxReturnIncome''
+                from
+                    sims.applications a
+                where
+                    a.application_number = applications.application_number
+                order by
+                    id desc
+                limit
+                    1
+            ) as "Student Income (from last edit)",
+            (
+                select
+                    a.data ->> ''taxReturnIncome''
+                from
+                    sims.applications a
+                where
+                    a.application_number = applications.application_number
+                order by
+                    id desc
+                limit
+                    1 offset 1
+            ) as "Student income (from two edits ago)",
```

Comment:
That is one scenario where I believe that we should provide more raw data and let the business do these types of calculations. For instance, instead of restricting the `application_status <> 'Overwritten'` we can return them all and let the data filtering happen in the CSV.
@HRAGANBC any thoughts on this?
@andrepestana-aot does the business provided any input on this?

Reply:
The input from the business is that we should query only the submitted applications. 
I believe that this report shouldn't be extracted for every history record because as it was discussed in the grooming, it will used by the ministry staff to check suspicious applications and I'm not sure if the staff will be wiling to filter and work on that rawness of data. @HRAGANBC ? Thoughts?

Reply:
I talked to @HRAGANBC and he said that it can have the whole history for the applications instead. I'm adding that to the next steps ticket.